content/_data/changelogs/weekly.yml

```diff
@@ -289,7 +289,8 @@
       message: >
         Packaging:
         Do not invoke recursive chown in JENKINS_HOME during the RPM post-install step.
-      issue: 23273
+      references:
+        - issue: 23273
```

Comment:
This change is unnecessary. It's only needed if the `references` section exists.